script.js

```diff
@@ -0,0 +1,10 @@
+
+var cells = document.getElementsByClassName("zero");
+var cells = document.getElementsByClassName("one");
+var cells = document.getElementsByClassName("two");
+var cells = document.getElementsByClassName("three");
+var cells = document.getElementsByClassName("zero");
+var cells = document.getElementsByClassName("zero");
+var cells = document.getElementsByClassName("zero");
+var cells = document.getElementsByClassName("zero");
+var cells = document.getElementsByClassName("zero");
```

Comment:
I think you probably had more code than this, but deleted it. Don't delete it. When it comes to this class, including incomplete/malfunctioning code is infinitely better than including a tiny bit of functioning code.